### ISSUE TITLE
Jasper can't find saved model

### ISSUE THREAD

**Issue by akafen** [TensorSpeech/TensorFlowASR#221]
hi, thank you developed this great toolkit and open source.but i have some problem and cant solve by myself.

i run train.py in examples/japer.After 6 epoches, loss reduce to 0.69. But I don't find any saved model in my saved directory

i think i was follow the instruction in readme.

python version:3.7.3
tf version: 2.3.2
ctc_loss in tf
use characters

config.yml:

> speech_config:
>   sample_rate: 16000
>   frame_ms: 25
>   stride_ms: 10
>   num_feature_bins: 80
>   feature_type: log_mel_spectrogram
>   preemphasis: 0.97
>   normalize_signal: True
>   normalize_feature: True
>   normalize_per_feature: False
> 
> decoder_config:
>   vocabulary: null
>   blank_at_zero: False
>   beam_width: 500
>   lm_config:
>     model_path: null
>     alpha: 2.0
>     beta: 1.0
> 
> model_config:
>   name: jasper
>   dense: True
>   first_additional_block_channels: 256
>   first_additional_block_kernels: 11
>   first_additional_block_strides: 2
>   first_additional_block_dilation: 1
>   first_additional_block_dropout: 0.2
>   nsubblocks: 3
>   block_channels: [256, 384, 512, 640, 768]
>   block_kernels: [11, 13, 17, 21, 25]
>   block_dropout: [0.2, 0.2, 0.2, 0.3, 0.3]
>   second_additional_block_channels: 896
>   second_additional_block_kernels: 1
>   second_additional_block_strides: 1
>   second_additional_block_dilation: 2
>   second_additional_block_dropout: 0.4
>   third_additional_block_channels: 1024
>   third_additional_block_kernels: 1
>   third_additional_block_strides: 1
>   third_additional_block_dilation: 1
>   third_additional_block_dropout: 0.4
> 
> learning_config:
>   train_dataset_config:
>     use_tf: True
>     data_paths:
>       - /home/liuyijiao/tensorflow_asr_datasets/tensorflow_asr.tsv
>     tfrecords_dir: /home/liuyijiao/tensorflow_asr_datasets/tfrecords
>     shuffle: True
>     cache: True
>     buffer_size: 128
>     drop_remainder: True
>     stage: train
> 
>   eval_dataset_config:
>     use_tf: True
>     data_paths:
>       - /home/liuyijiao/tensorflow_asr_datasets/valid_transcripts.tsv
>     tfrecords_dir: /home/liuyijiao/tensorflow_asr_datasets/tfrecords
>     shuffle: False
>     cache: True
>     buffer_size: 100
>     drop_remainder: True
>     stage: eval
> 
>   test_dataset_config:
>     use_tf: True
>     data_paths:
>       - /home/liuyijiao/tensorflow_asr_datasets/test_transcripts.tsv
>     tfrecords_dir: /home/liuyijiao/tensorflow_asr_datasets/tfrecords
>     shuffle: False
>     cache: True
>     buffer_size: 100
>     drop_remainder: True
>     stage: test
> 
>   optimizer_config:
>     class_name: adam
>     config:
>       learning_rate: 0.0001
> 
>   running_config:
>     batch_size: 32
>     num_epochs: 6
>     checkpoint:
>       filepath: /home/liuyijiao/tensorflow_asr_datasets/models/jasper/checkpoints/{epoch:02d}.h5
>       save_best_only: True
>       save_weights_only: False
>       save_freq: epoch
>     states_dir: /home/liuyijiao/tensorflow_asr_datasets/models/jasper/states
>     tensorboard:
>       log_dir: /home/liuyijiao/tensorflow_asr_datasets/models/jasper/tensorboard
>       histogram_freq: 1
>       write_graph: True
>       write_images: True
>       update_freq: epoch
>       profile_batch: 2
>

**Comment by usimarit** [comment]
I’ll close the issue here due to inactivity. Feel free to reopen if you have further questions.
